src/Token/AccessToken.php

```diff
@@ -34,11 +34,11 @@ class AccessToken
      */
     public function __construct(array $options = null)
     {
-        if (! isset($options['access_token'])) {
-            throw new \InvalidArgumentException(
-                'Required option not passed: access_token'.PHP_EOL
-                .print_r($options, true)
-            );
+        if (empty($options['access_token'])) {
+            throw new InvalidArgumentException(sprintf(
+                'Required option not passed: "%s"',
+                'access_token'
```

Comment:
I don't _need_ to, but I prefer the way it reads and it makes it easier to copy the exception later. I can actually update this PR to use the new `RequiredParameterTrait` and then it will make more sense.